Sources/MapboxMaps/Foundation/MapTransformDelegate.swift

```diff
@@ -0,0 +1,31 @@
+internal protocol MapTransformDelegate: AnyObject {
+    /// Gets the size of the map in points
+    var size: CGSize { get set }
```

Comment:
on Android, the size in MapTransformDelegate is read only, it is set through `MapboxRenderer`.

Reply:
We're ok here, as this property is currently internal.